.eslintrc.yml

```diff
@@ -22,6 +22,7 @@ overrides:
       - karma.conf.js
       - .wallaby.js
       - bin/*
+      - lib/*
```

Comment:
Don't think this is correct. You're now making everything ES6 rather than just the Node-specific JS.

Reply:
Well all the js at top level of `lib` (i.e. not browser/interfaces/reporters/cli). This was due to lint error using template literals. 
Do you think its safer just to add the 3 files using it here?